### DISCUSSION TITLE
https://issues.redhat.com/browse/RESTEASY-3495

### DISCUSSION THREAD

**Discussion by ericossj** [resteasy/resteasy#4368] [Q&A]
https://issues.redhat.com/browse/RESTEASY-3495

Hello everyone,
I have just tested with the WildFly server 34.0.0.Final (WildFly Core 26.0.1.Final) and
with RESTEasy version 7.0.0.Alpha3
however the problem of uploading files with accents continues to occur, any idea when it will be fixed?

**Comment by jamezp** [comment]
@ericossj Sorry, this appears to be stalled for some reason at the moment. I can try to have a look at it this week. I apologize for the delay.

One minor note, if you use 7.0 in WildFly make sure you're using WildFly Preview as 7.0 is a Jakarta REST 4.0 implementation.

**Comment by ericossj** [comment]
Sorry, I made a mistake,
I'm using RESTEasy version 6.2.10. Final

I'm just not able to update since Wildlfy version 31, because of this bug due to accentuation

**Comment by jamezp** [comment]
@ericossj I've added a reproducer to https://issues.redhat.com/browse/RESTEASY-3495. Is there any chance you could try that?

Simply use `mvn clean package`, then execute it with `java -jar target/standalone-multipart.jar /path/to/file.png`

**Comment by ericossj** [comment]
Good afternoon James,
I tried to run the command by passing a file name with an accent, and I got this error:

![image](https://github.com/user-attachments/assets/318ae216-8593-469b-a024-394ef9274a3a)

![image](https://github.com/user-attachments/assets/eedcc0d8-829d-489e-a273-fb3924ebcce9)

**Comment by ericossj** [comment]
running command passing a file without accent, it works:

![image](https://github.com/user-attachments/assets/a593df36-4891-447d-9b2f-99ca54288c13)

**Comment by ericossj** [comment]
when i try to upload through Wildfly 34 with Resteasy 6.2.10.Final it doesn't work

![image](https://github.com/user-attachments/assets/eecd1ebf-c249-4812-bbf8-70f553cda4ba)

debugging in eclipse, we have these objects constructed incorrectly because of the accent

![image](https://github.com/user-attachments/assets/50fc5129-e055-4793-994e-da0d1410c63d)

**Comment by jamezp** [comment]
Ah-ah, I need to test this on Windows. I've been testing on Linux.

**Comment by ericossj** [comment]
our server runs on linux platform,
and the problem happens James

**Comment by ericossj** [comment]
I updated in Jira
https://issues.redhat.com/browse/RESTEASY-3495
